core/jobs/batch_jobs/firebase_sync_jobs_test.py

```
@@ -0,0 +1,125 @@
+                PermissionError, 'must never be run in production'
+            ),
+        ):
+            self.assert_job_output_is_empty()
+
+    def test_run_with_existing_data_deletes_and_imports_records(self) -> None:
+        self.firebase_sdk_stub.create_user(
+            uid='old_fb', email='old@old.com', disabled=False
+        )
+
+        old_user = self.firebase_sdk_stub.get_user('old_fb')
+        self.assertEqual(old_user.uid, 'old_fb')
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)